charts/asset-manager/templates/clamd-configmap.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-clamd-conf
+  labels:
+    {{- include "asset-manager.labels" . | nindent 4 }}
+    app.kubernetes.io/component: web
```

Comment:
Probably want this to be something other than web. `clam`?

Reply:
thanks, I will fix